### PR TITLE
Experiment/ Show account details and tokens in configuration

### LINKED ISSUE #372: Show account details and tokens in configuration
There is no place in our configuration section where we show the account id and the associated tokens. This is often a pain point while debugging as the only place to see this information is the `core_config_data` table.

Showing each of the tokens and the account-id in the configuration section would make it easier to troubleshoot issues.

### PR DESCRIPTION
**Pull request by olsiqose-nosto**
## Description
Show the tokens in the Nosto Configuration panel

## Related Issue
closes #372

## Motivation and Context
Displaying the tokens will make the debuging process easier

## How Has This Been Tested?
Tested manually to display the right strings

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] I have assigned the correct milestone or created one if non existent.
- [ ] I have correctly labeled this pull request.
- [ ] I have linked the corresponding issue in this description.
- [ ] I have updated the corresponding Jira ticket.
- [ ] I have requested a review from at least 2 reviewers

### CODE REVIEW

File: etc/adminhtml/system.xml [hunk 1]
```diff
@@ -239,6 +239,14 @@
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                 </field>
             </group>
+            <group id="nosto_account_info" translate="label" type="text" sortOrder="60" showInDefault="0"
+                   showInWebsite="0" showInStore="1">
+                <label>Account Details and Tokens</label>
+                <field id="nosto_account_info_fields" translate="label comment" type="text" sortOrder="20"
+                       showInDefault="1" showInWebsite="1" showInStore="1">
```
Comment (mridang) [inline etc/adminhtml/system.xml]: Why would we show in website and default when tokens are only persisted on a store-level?

File: Block/Adminhtml/Form/Field/Tokens.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+    /**
+     * Get the Nosto account details
+     *
+     * @return \Nosto\Object\Signup\Account|null
+     */
+    public function getAccountDetails()
+    {
+        $id = (int) $this->request->getParam('store');
```
Comment (mridang) [inline Block/Adminhtml/Form/Field/Tokens.php]: Should this be `ScopeInterface::SCOPE_STORE`?

File: view/adminhtml/templates/tokens.phtml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                </td>
+            </tr>
+
+        <?php foreach($this->getAccountDetails()->getTokens() as $token): ?>
+            <tr>
+                <td class="token-title">
+                    <?php echo $token->getName() ?> Token
+                </td>
+                <td class="token-container">
+                    <?php echo $token->getValue() ?>
```
Comment (mridang) [inline view/adminhtml/templates/tokens.phtml]: Should this be wrapped in a <pre></pre> tag?

File: etc/adminhtml/system.xml (hunk 1 above)
Comment (olsiqose-nosto) [reply etc/adminhtml/system.xml]: They are momentally dependent on the parent group configuration.

File: view/adminhtml/templates/tokens.phtml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+            <tr>
+                <td class="token-title">
+                    <?php echo $token->getName() ?> Token
+                </td>
+                <td class="token-container" style="padding: 2.2rem 1.5rem 1rem 1.5rem;">
+
+                    <input type="text" readonly style="width:100%; word-wrap:break-word;"
+                        value="<?php echo $token->getValue() ?>"
+                    />
+
```
Comment (mridang) [inline view/adminhtml/templates/tokens.phtml]: Remove useless whitespace.

File: Block/Adminhtml/Form/Field/Tokens.php [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+        NostoHelperAccount $nostoHelperAccount,
+        array $data = []
+    ) {
+        parent::__construct($context, $data);
+        $this->nostoHelperAccount = $nostoHelperAccount;
+        $this->nostoHelperScope = $nostoHelperScope;
+        $this->request = $request;
+    }
+
+    //@codingStandardsIgnoreStart
```
Comment (supercid) [inline Block/Adminhtml/Form/Field/Tokens.php]: Why is this whole file being ignored?

**Pull request closed without merging**